### PR TITLE
Implemented the Gnome Sort in Java, Python, C, C++.

### LINKED ISSUE #1831: Implementation of Gnome Sort
I want to work on it in C, C++, Java, and Python. Please assign this issue to me @singh-shreya6 @raghav-dalmia @asha15

### PR DESCRIPTION
**Pull request by srimani-programmer**
Fixes #1831

#### Checklist:
- [x] 4 space indentation.

- [x] Coding conventions are followed.
- [ ] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Implemented the Gnome Sorting algorithm in 4 languages with a complete explanation.

#### Languages Used:
-  Java
-  Python3
- C++
- C

#### Files Added:
- Gnome_Sort.Java
- Gnome_Sort.py
- Gnome_Sort.cpp
- Gnome_Srot.c
- README.md

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by srimani-programmer** [comment]
A small mistake made while pushing the code I want to Push in proper manner.

**Comment by somya-kapoor** [comment]
Please mention the issue number you are trying to fix.

**Comment by srimani-programmer** [comment]
The issue number is #1831. Sorry for the improper communication.

**Comment by raghav-dalmia** [comment]
Also remove readme file from your PR.

File: Gnome_Sort/Gnome_Sort.c [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#include <stdio.h> 
+ 
+  
+// A function which is used to sort the algorithm using gnome sort 
+void gnome_sort(int arr[], int n) 
+{   
+    // 
+        int index = 0;
```
Comment (jainaman224) [inline Gnome_Sort/Gnome_Sort.c]: Incorrect indentation.

File: Gnome_Sort/Gnome_Sort.c [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    int arr[] = {3, 2, 1, -5, 7}; 
+    int n = sizeof(arr) / sizeof(arr[0]); 
+    gnome_sort(arr, n); 
+
+    for(int i=0;i<n;i++){
+        printf("%d\n",arr[i]);
+    }
+  
+    return 0; 
+}
```
Comment (jainaman224) [inline Gnome_Sort/Gnome_Sort.c]: Add sample output.

File: Gnome_Sort/Gnome_Sort.c [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+    }
+   
+} 
+  
+  
+// Sample Test cases to test above function. 
+int main() 
+{ 
+    int arr[] = {3, 2, 1, -5, 7};
```
Comment (jainaman224) [inline Gnome_Sort/Gnome_Sort.c]: Take input from user.

File: Gnome_Sort/Gnome_Sort.c (hunk 2 above)
Comment (jainaman224) [reply Gnome_Sort/Gnome_Sort.c]: Please make all suggested changes to all files.

**Comment by srimani-programmer** [comment]
Accidentally Closed the PR I am not able to open it can you please Help it How to reopen the PR.

**Comment by srimani-programmer** [comment]
Opened the new PR Sir. No problem I hope

**Pull request closed without merging**